aws/sdk/integration-tests/s3/tests/customizable-operation.rs

```diff
@@ -57,3 +57,59 @@ async fn test_s3_ops_are_customizable() {
         snapshot_signature
     );
 }
+
+#[cfg(aws_sdk_orchestrator_mode)]
+#[tokio::test]
+async fn test_extract_metadata_via_customizable_operation() {
```

Comment:
This should be a Kotlin unit test for `MetadataCustomization`.

Reply:
Thanks for bringing this up! I was actually on the fence about this. I originally wrote this test as a Kotlin test but then thought that writing a test fully in a Rust source file could better serve as documentation for the customers who needs to upgrade their code, instead of Rust snippets embedded in a Kotlin file. But I suppose they have no trouble following the Rust code even in a Kotlin unit test and more importantly, from an engineering process point of view, having a Kotlin unit test is better than having it in `aws/sdk/integration-tests`.

Reply:
Addressed in 9472734.